pom.xml

```diff
@@ -1246,7 +1247,13 @@
             <dependency>
                 <groupId>org.testcontainers</groupId>
                 <artifactId>testcontainers</artifactId>
-                <version>1.12.3</version>
+                <version>${dep.testcontainers.version}</version>
```

Comment:
FYI there is `testcontainers-bom` that lists all modules

Reply:
@bsideup Thanks. Updated.